mobile/app/utils/googleAuth.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  GoogleSignin,
+  statusCodes,
+} from "@react-native-google-signin/google-signin";
+import { supabase } from "./supabase";
+
+export const googleAuth = async () => {
+  GoogleSignin.configure({
+    scopes: ["https://www.googleapis.com/auth/drive.readonly"],
+    iosClientId:
+      "865962598053-cpic88pj6c8raaqlsca0qhua9mk1id7c.apps.googleusercontent.com",
```

Comment:
```suggestion
      process.env.EXPO_PUBLIC_IOS_CLIENT_ID,
```
Can we move this id to env please?